README.md

```diff
@@ -164,136 +166,168 @@ Supported domains: BLAS, LAPACK, RNG, DFT, SPARSE_BLAS
-            <td align="center">LLVM*</td>
         </tr>
         <tr>
             <td rowspan=4 align="center">RNG</td>
             <td align="center">x86 CPU</td>
-            <td rowspan=2 align="center">Intel(R) oneAPI Math Kernel Library</td>
+            <td align="center">Intel(R) oneMKL</td>
+            <td align="center">Intel DPC++</br>Open DPC++</br>hipSYCL</td>
             <td align="center">Dynamic, Static</td>
-            <td align="center">DPC++, LLVM*, hipSYCL</td>
         </tr>
         <tr>
             <td align="center">Intel GPU</td>
+            <td align="center">Intel(R) oneMKL</td>
+            <td align="center">Intel DPC++</td>
             <td align="center">Dynamic, Static</td>
-            <td align="center">DPC++</td>
         </tr>
         <tr>
             <td align="center">NVIDIA GPU</td>
             <td align="center">NVIDIA cuRAND</td>
+            <td align="center">Open DPC++</br>hipSYCL</td>
             <td align="center">Dynamic, Static</td>
-            <td align="center">LLVM*, hipSYCL</td>
         </tr>
         <tr>
             <td align="center">AMD GPU</td>
             <td align="center">AMD rocRAND</td>
+            <td align="center">Open DPC++</br>hipSYCL</td>
             <td align="center">Dynamic, Static</td>
-            <td align="center">LLVM*, hipSYCL</td>
         </tr>
         <tr>
-            <td rowspan=5 align="center">DFT</td>
-            <td align="center">Intel GPU</td>
-            <td rowspan=2 align="center">Intel(R) oneAPI Math Kernel Library</td>
+            <td rowspan=8 align="center">DFT</td>
+            <td rowspan=2 align="center">x86 CPU</td>
+            <td align="center">Intel(R) oneMKL</td>
+            <td align="center">Intel DPC++</td>
             <td align="center">Dynamic, Static</td>
-            <td align="center">DPC++</td>
         </tr>
         <tr>
-            <td align="center">x86 CPU</td>
+            <td align="center">portFFT (<a href="https://github.com/codeplaysoftware/portFFT#supported-configurations">limited API support</a>)</td>
+            <td align="center">Intel DPC++</td>
             <td align="center">Dynamic, Static</td>
-            <td align="center">DPC++</td>
         </tr>
         <tr>
-            <td align="center">NVIDIA GPU</td>
+            <td rowspan=2 align="center">Intel GPU</td>
+            <td align="center">Intel(R) oneMKL</td>
+            <td align="center">Intel DPC++</td>
+            <td align="center">Dynamic, Static</td>
+        </tr>
+        <tr>
+            <td align="center">portFFT (<a href="https://github.com/codeplaysoftware/portFFT#supported-configurations">limited API support</a>)</td>
+            <td align="center">Intel DPC++</td>
+            <td align="center">Dynamic, Static</td>
+        </tr>
+        <tr>
+            <td rowspan=2 align="center">NVIDIA GPU</td>
             <td align="center">NVIDIA cuFFT</td>
+            <td align="center">Open DPC++</td>
```

Comment:
Should work with icpx with Codeplay's oneAPI for NVidia GPUs.

Reply:
@hjabird  Can you please explain what you mean and suggest with your comment? 
Are you suggesting to add a new entry to the supported compilers? If yes, that needs to be added to our CI before claiming support for it.

Reply:
You should be able to compile the *cuFFT* and *rocFFT* DFT backends using the Intel release of the oneAPI base toolkit (which includes *icpx*), and Codeplay's releases of the [oneAPI for Nvidia GPUs](https://developer.codeplay.com/products/oneapi/nvidia/2024.0.2/guides/) and [oneAPI for AMD GPUs](https://developer.codeplay.com/products/oneapi/amd/home/) respectively. Its been what I've been using whenever I've build oneMKL for Nvidia or AMD backends, although perhaps I shouldn't have been doing this.

Perhaps for now we can say that using the Codeplay plugins should work but is not tested?

Reply:
I am not in favor of  just saying something without testing it in CI and/or providing clear descriptions of how to reproduce it.

Reply:
Are Codeplay's documentation [Install oneAPI for NVIDIA GPUs](https://developer.codeplay.com/products/oneapi/nvidia/2024.0.2/guides/get-started-guide-nvidia#install-oneapi-for-nvidia-gpus) and [Install oneAPI for AMD GPUs (beta)](https://developer.codeplay.com/products/oneapi/amd/2024.0.2/guides/get-started-guide-amd#install-oneapi-for-amd-gpus--beta-) sufficient for users to try out the plugins with `icpx`? We could encourage users to experiment with the plugins, then adding them in the supported configurations once we test them in CI.

Reply:
I'll open an issue to discuss this in, becuase I think I've sidetracked this PR enough (sorry @dnhsieh-intel!). I'll close the associated comments.